### PR TITLE
Add pg_mon extension

### PR DESCRIPTION
**Pull request by RafiaSabih**
This extension will add more details and help in improving query monitoring. For more information check [here](https://github.com/RafiaSabih/pg_mon).

### CODE REVIEW

File: postgres-appliance/Dockerfile [hunk 1]
```diff
@@ -70,6 +70,7 @@ ARG DEB_PG_SUPPORTED_VERSIONS="$PGOLDVERSIONS $PGVERSION"
 ENV POSTGIS_VERSION=3.0 \
     BG_MON_COMMIT=7a4a4a621f89cc032bd7d497ce2373a1220c89a5 \
     PG_AUTH_MON_COMMIT=902c670c9f2071b30e1d4437e7a38e2e136f9be6 \
+    PG_MON_COMMIT=e4b577c71583cbdbc1f1c1cdf2612bc58ada5288= \
```
Comment (CyberDem0n) [inline postgres-appliance/Dockerfile]: trailing `=`

File: postgres-appliance/Dockerfile [hunk 2]
```diff
@@ -123,6 +124,7 @@ RUN export DEBIAN_FRONTEND=noninteractive \
 \
     && curl -sL https://github.com/CyberDem0n/bg_mon/archive/$BG_MON_COMMIT.tar.gz | tar xz \
     && curl -sL https://github.com/RafiaSabih/pg_auth_mon/archive/$PG_AUTH_MON_COMMIT.tar.gz | tar xz \
+    && curl -sL https://github.com/RafiaSabih/pg_mon/archive/$PG_MON_COMMIT.tar.gz | tar xz \
```
Comment (CyberDem0n) [inline postgres-appliance/Dockerfile]: Lets move it [here](https://github.com/zalando/spilo/pull/420/files#diff-82f985fcf1262fb2d3d1843b9e74d9c3R109)

File: postgres-appliance/Dockerfile [hunk 3]
```diff
@@ -214,7 +216,7 @@ RUN export DEBIAN_FRONTEND=noninteractive \
                 EXTRA_EXTENSIONS=""; \
             fi \
 \
-            && for n in bg_mon-${BG_MON_COMMIT} pg_auth_mon-${PG_AUTH_MON_COMMIT} set_user $EXTRA_EXTENSIONS; do \
+            && for n in bg_mon-${BG_MON_COMMIT} pg_auth_mon-${PG_AUTH_MON_COMMIT} pg_mon-${PG_MON_COMMIT} set_user $EXTRA_EXTENSIONS; do \
```
Comment (CyberDem0n) [inline postgres-appliance/Dockerfile]: Lets move it [here](https://github.com/zalando/spilo/pull/420/files#diff-82f985fcf1262fb2d3d1843b9e74d9c3R211)

**Comment by CyberDem0n** [comment]
:+1:

**Comment by RafiaSabih** [comment]
:+1:

**Comment by FxKu** [comment]
:+1:

**Comment by RafiaSabih** [comment]
:+1

**Comment by RafiaSabih** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Comment by RafiaSabih** [comment]
:+1:

**Comment by erthalion** [comment]
:+1:

**Pull request merged**